app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,62 @@
+class Admin::UsersController < ApplicationController
+  before_action :set_user, only: [:edit, :update, :destroy, :show]
+  before_action :login_required
+  before_action :admin_required
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to admin_users_url, success: "「#{@user.name}」#{t("view.flash.create_message")}"
+    else
+      flash.now[:danger] = "#{t("view.flash.create_alert")}"
+      render :new
+    end
+  end
+
+  def edit
+  end
```

Comment:
こちらも1行で記載できます。
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/EmptyMethod
```suggestion
  def edit; end
```

Reply:
d9026e4 で修正しました。